### DISCUSSION TITLE
How to ask for help about licenses and purchase?

### DISCUSSION THREAD

**Discussion by jcfrog** [aclap-dev/video-downloadhelper#2] [License & Purchase requests]
For any issue about purchases and licenses, it is not the place to share private info.

Please [go there](https://github.com/aclap-dev/video-downloadhelper/wiki/I-need-help#license-problem).

**Comment by mind2019** [comment]
Hola, compré una licencia y me muestra "Licencia bloqueada". ¿Qué debo hacer?

**Reply by paulrouget** [reply]
You should have received an email. Otherwise, please go here: https://github.com/aclap-dev/video-downloadhelper/wiki/I-need-help#license-problem
